build-output-api/isr-api-routes/.vercel/output/functions/index.func/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+
+const clientJS = fs.readFileSync(__dirname + '/client.js').toString();
+
+module.exports = (req, res) => {
+  res.setHeader('Content-Type', 'text/html; charset=utf-8')
+
+  // NOTE: the below revalidate link is meant for demo purposes only
+  // actual authentication should be used in actual projects
+  res.end(`
+    <h1>On-Demand Incremental Static Regeneration (ISR) with API Routes Example</h1>
+    <p>The "API Result" below will only get updated after you click the Revalidate link.</p>
+
+    <p>API Result: <span id="result"></span></p>
+
+    <p><a href="/revalidate?authToken=a13f94f6-a441-47ca-95fc-9f44f3450295">Revalidate</a></p>
```

Comment:
Do we want to add a comment here saying that a real application wouldn't send the auth token like this?